.gitignore

```diff
@@ -3,3 +3,6 @@ dist
 gulpfile.js
 node_modules
 manifest.json
+.idea/
+.vscode/
+dotfiles/
```

Comment:
I also updated the .gitignore file in order to avoid committing these unwanted files.